vxfreeswitch/voice.py

```diff
@@ -494,6 +495,7 @@ def dial_outbound(self, to_addr):
             call_uuid = reply.args[1]
         if self.config.wait_for_answer:
             self._unanswered_channels[call_uuid] = Deferred()
+        self._msisdn_mapping[call_uuid] = to_addr
```

Comment:
is this the `call_uuid` the `vumi_message_id`? If so, are we sure that is what the `client.get_address()` call returns? I thought it returned the uuid of the call session?